src/status.cpp

```diff
@@ -53,6 +53,8 @@ const std::map<const StatusCode, const std::string> Status::statusMessageMap = {
     {StatusCode::SEQUENCE_ALREADY_EXISTS, "Sequence with provided ID already exists"},
     {StatusCode::SEQUENCE_ID_NOT_PROVIDED, "Sequence ID has not been provided in request inputs"},
     {StatusCode::INVALID_SEQUENCE_CONTROL_INPUT, "Unexpected value of sequence control input"},
+    {StatusCode::SEQUENCE_ID_BAD_TYPE, "Sequence ID has been provided with bad proto type"},
+    {StatusCode::SEQUENCE_CONTROL_INPUT_BAD_TYPE, "Sequence control input was provided with bad proto type"},
```

Comment:
It may be misleading since we get the same code when those values are not passed at all. So maybe a better message would be something like:
"Could not find sequence id / sequence control input in expected tensor proto field - uint64_val / uint32_val" 
This would cover both situations - when value is not provided at all and when it's been written to unexpected field like tensor_content.